MediaGalleryUi/etc/db_schema_whitelist.json

```diff
@@ -24,4 +24,4 @@
             "MEDIA_GALLERY_ASSET_GRID_ID_MEDIA_GALLERY_ASSET_ID": true
         }
     }
-}
+}
```

Comment:
I'd keep that space at the end of the file as it's readded by PHP with default settings automatically

Reply:
This change added automatically by the CLI Magento command.